packages/analytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.3.2
+
+-   npm-prerelease: @trezor/analytics 1.3.2-beta.1 (da51b9cf12)
```

Comment:
I would say this type of commit shouldn't be here, but let's deal with this later sometime

Reply:
Yes, I know this type of changelog does not add much value. I am in for improving it but this is actually the only commit that was generated to this package. And the package requires to be updated becuase it's dependency `@trezor/utils` was updated and therefore this package has to be released as well. 

This kind of conclusions are very difficult to create in an automate way. And since our focus is to create a good changelog for `@trezor/connect` so we decided to leave this kind of changelogs like that.

But I am happy to listen other ideas.

Reply:
Maybe one easy and straight forward idea could be to not include the commits starting with ` npm-prerelease` But then it would be nice to add something not only the version number.

Maybe we can assume that it was updated because a dependency was updated if there are no commits and add a message saying something like: `Releasing due to dependencies update`.

WDYT? :question: